### DISCUSSION TITLE
WSA Sideloader don't open up ADB prompt

### DISCUSSION THREAD

**Discussion by Hom3r86** [infinitepower18/WSA-Sideloader#97] [Support]
I can't use Android anymore. When i'm activating devmode, Windows shows me the ADB IP "127.0.0.1:0".
Since i've got this problem, i'm not able to use WSA Sideloader anymore or any other app.

**Comment by infinitepower18** [comment]
You can change the ADB address in WSA Sideloader via the settings page. Strange the ADB IP is on port 0 though.

**Reply by Hom3r86** [reply]
I did this but it didn't solve my problem :( I have reinstalled everything but it still doesnt work. I think only a new windows installation will help -.-'

**Reply by Hom3r86** [reply]
<img width="456" alt="image" src="https://github.com/infinitepower18/WSA-Sideloader/assets/143504478/0b472b0d-a698-49e7-86ca-22e151f71337">

I'm stuck here, after a few seconds the window dissappears

**Reply by infinitepower18** [reply]
Are you using the WSA from Microsoft Store?

**Reply by Hom3r86** [reply]
No it's not available for germany. But i've downloaded it from https://store.rg-adguard.net

I don't want to bother you further because that's my problem to solve. But thank you for your fast responses
